playbooks/vars/maas-auth.yml

```diff
@@ -0,0 +1,11 @@
+maas_username: "admin"
+
+maas_api_key: "None"
+
+maas_auth_url: "https://identity.api.rackspacecloud.com/v2.0"
+
+maas_auth_method: "password"
+
+maas_auth_token: "None"
```

Comment:
I see above we have `-maas_auth_token: "{{ lookup('env', 'MAAS_AUTH_TOKEN') | default(null) }}"`, and I know we feed MAAS_AUTH_TOKEN in from jenkins.